docs/current_docs/integrations/php.mdx

```diff
@@ -7,7 +7,17 @@ import TabItem from '@theme/TabItem';
 
 # PHP
 
-Dagger can be used to perform common CI tasks - testing, containerizing, publishing and more - for any PHP application.
+Dagger can be used to perform common CI tasks - testing, containerizing, publishing and more - for any PHP application. If programing in PHP is your favorite, you are in luck. A group of Daggernauts have created an experimental Dagger PHP SDK for you to use. 
+
+# How it Works
+
+Before you get started with the experimental SDK, we suggest that you try out our [quickstart](https://docs.dagger.io/quickstart) first. 
+
+# Prerequisites
+
+NEED TO FILL OUT --- What should be already installed and configured? Links to third-party install/configuration docs.
```

Comment:
The SDK module runtime takes care of all the dependencies, so really the only thing that is needed is the dagger cli tool, engine and docker which would be taken care of in the quickstart guide. 

A good PHP IDE would be helpful, but not required.